homeassistant/components/zwave_me/const.py

```diff
@@ -12,11 +12,14 @@
     "doorlock",
     "sensorMultilevel",
     "sensorBinary",
+    "switchRGBW",
+    "switchRGB",
```

Comment:
I'd define constants for these and reuse them in the different platforms in the dispatcher signal.